From: Jason Rhinelander <jason@imaginary.ca>
Date: Thu, 25 Jul 2019 21:16:24 -0300
Subject: Reduce required cmake version

It looks (from the comment) like this requirement is only actually
needed for android.
---
 vendor/nlohmann/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vendor/nlohmann/CMakeLists.txt b/vendor/nlohmann/CMakeLists.txt
index 1de729a..48d130d 100644
--- a/vendor/nlohmann/CMakeLists.txt
+++ b/vendor/nlohmann/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.6)
+cmake_minimum_required(VERSION 3.5.1)
 
 ##
 ## PROJECT
